tests/e2e/replication_slot_test.go

```diff
@@ -48,6 +50,19 @@ var _ = Describe("Replication Slot", func() {
 	})
 
 	It("can manage Replication slots for HA", func() {
+		// check if current postgres version is pg 11
+		if strings.Contains(os.Getenv("POSTGRES_IMG"), ":10") {
+			AssertCreateNamespace(namespace, env)
```

Comment:
This should be factored together with the creation on line 67

Reply:
Hm. This is slightly different. The AssertCreateCluster already includes waiting for the namespace.
However, even if there is some functional redundancy, no harm done, will do as you suggest.
In a future ticket I'd like to refactor the AssertCreateCluster to assume the namespace is already there.

Reply:
I would just use AssertCreateCluster also in the other place. It doesn't harm to wait until the object is visible.

Reply:
Yep

Reply:
Better yet, do it in the beginning of the test. Either pg < 11 or >= 11 need it created.